### ISSUE TITLE
[DOC] - Customizing Ashpy Examples

### ISSUE THREAD

**Issue by mr-ubik** [zurutech/ashpy#50]
- [ ] Add documentation and examples for the implementation of custom `Metric`.
	- [ ] How to convert a `keras.metrics.Metric` into an Ashpy one?
- [ ] Add documentation and examples for the implementation of custom `Callbacks`.
